application-admintools-ui/src/main/resources/AdminTools/Code/Macros.xml

```diff
@@ -152,5 +152,54 @@
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+#macro (displayPagesOverNumberOfComments $maxComments)
+  #set ($discard = $xwiki.jsfx.use('js/xwiki/table/tablefilterNsort.js', true))
+  &lt;div class="modal fade" id="pagesOverNumberOfComments" tabindex="-1" role="dialog"
+    aria-labelledby="pagesOverNumberOfCommentsLabel"&gt;
+    &lt;div class="modal-dialog" role="document"&gt;
+      &lt;div class="modal-content"&gt;
+        &lt;div class="cache-modal-content"&gt;
+          &lt;div class="modal-header"&gt;
+            &lt;button type="button" class="close" data-dismiss="modal" aria-label="Close"&gt;
+              &lt;span aria-hidden="true"&gt;&amp;times;&lt;/span&gt;&lt;/button&gt;
+            &lt;h1 class="modal-title"&gt;
+              $escapetool.xml($services.localization.render('adminTools.dashboard.spamPage.modal.title',
+              [$maxComments]))&lt;/h1&gt;
+          &lt;/div&gt;
+          &lt;div class="modal-body"&gt;
+            #set ($spamPages = $services.admintools.getPagesOverGivenNumberOfComments($maxComments))
+            #if ($spamPages.size() != 0)
+              &lt;table id="wikiComments" class="sortable doOddEven admin-tools-spam-pages"&gt;
```

Comment:
You do you need the id? Using a class seems enough

Reply:
From what I've experimented and seen, the id is needed for the table filters, otherwise they won't work.

Reply:
Ok, indeed
Then you should find a specific id, since this one is too generic. Maybe `adminTools_spamPages`?